exercises/03-keywords_and_variables.md

```diff
@@ -116,21 +127,34 @@ ${VARIABLE_NAME} =    variable value
 ```
```

Comment:
another `${VARIABLE_NAME}` above this line

why can't I edit the line directly, Github?! :(

Reply:
Changed.